### ISSUE TITLE
Support Custom Labels for Buttons

### ISSUE THREAD

**Issue by mstmustisnt** [marmelab/admin-on-rest#230]
It would be great if in such places as [this](https://github.com/marmelab/admin-on-rest/blob/master/src/mui/button/CreateButton.js#L8) you'd support some custom label
```
const CreateButton = ({ basePath = '', label = "Create" }) => <FlatButton
    primary
    label={label}
    icon={<ContentAdd />}
    containerElement={<Link to={`${basePath}/create`} />}
    style={{ overflow: 'inherit' }}
/>;
``` 
The same can be applied to all the other buttons (Edit, List etc)
It would help to create admins based on admin-on-rest in other than English languages.

I understand I can duplicate your code and use my own buttons. but still it would be more convenient to pass` label` in `props`

**Comment by fzaninotto** [comment]
Fixed in #234

**Comment by mstmustisnt** [comment]
thank you @fzaninotto
